apps/vaporgui/CMakeLists.txt

```diff
@@ -106,6 +105,42 @@ set (SRCS
 	VSection.h
 	VLineItem.cpp
 	VLineItem.h
+    VContainer.cpp
+    VContainer.h
+    VComboBox.cpp
+    VComboBox.h
+    VSpinBox.cpp
+    VSpinBox.h
+    VCheckBox.cpp
+    VCheckBox.h
+    VLineEdit.cpp
+    VLineEdit.h
+    VSlider.cpp
+    VSlider.h
+    VSliderEdit.cpp
+    VSliderEdit.h
+    VPushButton.cpp
+    VPushButton.h
+    QPushButtonWithDoubleClick.h
+    VFileSelector.cpp
+    VFileSelector.h
+    VFrame.h
+    VGeometry2.cpp
+    VGeometry2.h
+	PWidget.cpp
+	PWidget.h
+	PCheckbox.cpp
+	PCheckbox.h
+	PDoubleInput.cpp
+	PDoubleInput.h
+	PLineItem.cpp
+	PLineItem.h
+	PEnumDropdown.cpp
+	PEnumDropdown.h
+	PColorSelector.cpp
+	PColorSelector.h
+	PFileSelector.cpp
+	PFileSelector.h
```

Comment:
In order to raise a comment that could be resolved later, I leave it hear, though it has nothing to do with this CMakeList.txt file.

The random mode doesn't paint anything.

Reply:
Fixed